test/types/queries.test.ts

```diff
@@ -393,7 +393,8 @@ async function gh12342_manual() {
 
 async function gh12342_auto() {
   interface Project {
-    name?: string, stars?: number
+    name?: string | null,
+    stars?: number | null
```

Comment:
is adding `| null` now required or is this just to test this case? (because the `?` should already add `| undefined`)

Reply:
For this particular case, just for testing. If there are cases where you specifically define a type, like `const v: number | undefined = doc.stars`, or `function foo(): number | undefined { return doc.stars; }` then you will need to add `null`.